tiledb/api/src/query/strategy/mod.rs

```diff
@@ -0,0 +1,374 @@
+use std::collections::HashMap;
```

Comment:
I'm not sure why this is showing up as new, it was moved from `strategy.rs` to `strategy/mod.rs`

Reply:
Git's rename detection is all heuristics based and it gets things wrongs occasionally. No need to worry though because the rename detection doesn't actually do anything internally.